PowerUp/app/src/main/java/powerup/systers/com/IntroActivity.java

```diff
@@ -0,0 +1,64 @@
+package powerup.systers.com;
+
+import android.content.Intent;
+import android.content.SharedPreferences;
+import android.graphics.Color;
+import android.os.Bundle;
+
+import com.github.paolorotolo.appintro.AppIntro;
+import com.github.paolorotolo.appintro.AppIntroFragment;
+
+public class IntroActivity extends AppIntro {
+
+    private SharedPreferences settings;
+    private boolean isfirstRun;
+
+    @Override
+    public void init(Bundle savedInstanceState) {
+
+        settings = getSharedPreferences("prefs", 0);
+        isfirstRun = settings.getBoolean("firstRun", true);
+        if (isfirstRun) {
+            addSlide(new IntroFragment(R.layout.intro_screen_1));
```

Comment:
Instead of passing the resource id, I think it would be better if you pass a simple integer from 1 to 6 as this way the implementation of the fragment content will be owned by the IntroFragment and it will be transparent to this Activity